posts/2024-07-24_working__groups_.../working__groups__updates.qmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Working Groups Updates"
+author:
+  - name: pharmaverse council
+description: "An update from pharmaverse council around how packages will get included in pharmaverse in future."
+date: "2024-07-24"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Community]
+# feel free to change the image
+image: "pharmaverse.PNG"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-07-24_working__groups_..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+Hi community,
+
+We wanted to share with you an update from council discussions around our
+pharmaverse working groups. Up until now, these have been the decision-holders of
+which packages are included/excluded from the pharmaverse. Although open for
+all to join, they ended up being quite lean and likely not representative of the
+true depth of this community. So we’ve decided to open up the package decisions
+to our full community in future via our [Slack workspace](https://pharmaverse.slack.com/), to make for a more
+inclusive community where everyone has a voice. This is now updated on our
+website – see the FAQ section on the [homepage](https://pharmaverse.org/).
+
+![pharmaverse is ours!](hands.PNG){fig-align="center"}
+
+Moving forwards, instead of working groups we will maintain some
```

Comment:
Hi @rossfarrugia 
I don't quite understand what the council is now responsible for, if not deciding on which packages go into the pharmaverse. 
Is there a list of responsibilities? 
What do you mean exactly with: 

[...] we plan
to channel any working group type energy towards
[PHUSE DVOST](https://advance.phuse.global/pages/viewpage.action?pageId=327777)
given our strong partnership with PHUSE.

Does this mean that any ro most future work done by the council will be done to support DVOST or in consultation with DVOST?

Reply:
@StefanThoma the council always delegated the responsibility of package inclusion to the WGs which we are now disbanding in favour of letting the community decide. 

Council responsibilities are explained on a separate page: https://pharmaverse.org/contribute/council/

The council is fully committed to continue to grow and strengthen pharmaverse independent of any PHUSE support. The blog is therefore not changing anything in the council - it's about our WGs where we see more potential by utilising PHUSE DVOST and channeling energy that way instead of starting our own pharmaverse WGs to solve specific industry OS challenges. This was already happening really - as although we gave our WGs freedom to decide their own scope they all only focused on pharmaverse package inclusion, as any other wider OS topics already have industry WGs that could be joined and supported.

I'll add a note to the blog to reiterate that none of this impacts council.